test/e2e/integration/frontend-test/custom-button.ts

```diff
@@ -27,6 +29,10 @@ describe('Custom Button', () => {
     cy.goto('changename');
 
     cy.findByRole('textbox', { name: /Her kan man skrive input/ }).type('Generate frontend actions');
+
+    cy.waitUntilSaved();
+    cy.waitUntilNodesReady();
```

Comment:
Not sure if these will suffice. During the tests I ran, nodes and saving is already done, but the possible layout-shift that caused this flakiness seemed related to something else. These will definitely stall Cypress by just doing more, but not by much. I would have added a `cy.wait(500)` instead (and wrote a comment about why).

Reply:
Just had to test in the pipeline to see if it did anything 🙈  Waiting an arbitrary amount is brittle and is best case adding extra wait time, as you say. So would like to get to the bottom of why this happens.